src/components/FilePreview/FilePreview.tsx

```diff
@@ -0,0 +1,114 @@
+import React, {useEffect, useState} from 'react';
+import type {FC, MouseEventHandler} from 'react';
+import {block} from '../utils/cn';
+
+import {
+    FileZipper as ArchiveIcon,
+    Code as CodeIcon,
+    FileQuestion as DefaultIcon,
+    Picture as ImageIcon,
+    MusicNote as MusicIcon,
+    LogoAcrobat as PdfIcon,
+    LayoutHeaderCellsLarge as TableIcon,
+    TextAlignLeft as TextIcon,
+    Filmstrip as VideoIcon,
+} from '@gravity-ui/icons';
+import {Card, Icon, IconData, Text, Tooltip} from '@gravity-ui/uikit';
+
+import {getFileType} from './utils';
+import {FilePreviewAction, FilePreviewActionProps} from './FilePreviewAction';
+
+import './FilePreview.scss';
```

Comment:
i guess that should be the last import

Reply:
Before updating the eslint plugin, this is the correct code!

https://github.com/gravity-ui/components/pull/84